src/test/java/api/support/builders/LoanPolicyBuilder.java

```diff
@@ -591,6 +626,30 @@ public LoanPolicyBuilder withRecallsRecallReturnInterval(Period recallsRecallRet
       this.alternateFixedDueDateScheduleId,
       this.renewable,
       this.recallsMinimumGuaranteedLoanPeriod,
-      recallsRecallReturnInterval.asJson());
+      recallsRecallReturnInterval.asJson(),
+      this.alternateCheckoutLoanPeriod);
+  }
+
+  public LoanPolicyBuilder withAlternateCheckoutLoanPeriod(Period alternateCheckoutLoanPeriod) {
```

Comment:
I think it is important that this refers to the situation when it applies, in this case it is hold requests (admittedly, my suggestion has compromises due to length)

```suggestion
  public LoanPolicyBuilder withHoldRequestAlternateLoanPeriod(Period alternateLoanPeriod) {
```